en/frames_rover/ackermann_rover.md

```diff
@@ -0,0 +1,246 @@
+# Ackermann Rover
+
+<Badge type="tip" text="main (PX4 v1.16+)" /> <Badge type="warning" text="Experimental" />
+
+An _Ackermann Rover_ controls its direction by pointing the front wheels in the direction of travel — the [ackermann steering geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) compensates for the fact that wheels on the inside and outside of the turn move at different rates.
+This kind of steering is used on most commercial vehicles, including cars, trucks etc.
+
+![Traxxas Stampede VXL](../../assets/airframes/rover/traxxas_stampede_vxl/stampede.jpg)
+
+::: info
+The module does not require the ackermann geometry and will work with any front-steering rover.
+:::
+
+## Airframe Setup
+
+This section will go through the most basic setup that is necessary to start using the rover.
+
+1. To enable the module flash the PX4 rover build onto your flight controller.
+
+   ::: info
```

Comment:
I have pushed this info into firmware. It's clearly not recommended so we should make it clearly "a less preferred resort".

Reply:
Makes sense to me!